<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboardWindow</class>
 <widget class="QDialog" name="AdminDashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharityLink Admin Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="adminWelcomeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>900</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Welcome, Admin!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewUsersButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 18pt "Open Sans";
background-color: red;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>View All Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewDonationsButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 18pt "Open Sans";
background-color: red;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>View All Donations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewRequestsButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 18pt "Open Sans";
background-color: red;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>View All Requests</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAuditLogsButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 18pt "Open Sans";
background-color: red;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>View Audit Logs</string>
   </property>
  </widget>
  <!-- NEW BUTTON: Reverse Donation/Match -->
  <widget class="QPushButton" name="reverseButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 16pt "Open Sans";
background-color: rgb(150, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Reverse Donation/Match</string>
   </property>
  </widget>
  <!-- NEW BUTTON: Update Delivery Status -->
  <widget class="QPushButton" name="updateDeliveryButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font: 700 16pt "Open Sans";
background-color: rgb(0, 128, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Update Delivery Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>500</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 700 14pt "Open Sans";
background-color: rgb(220, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminNotificationLabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     background-color: rgb(255, 255, 200);
     border: 1px solid rgb(200, 200, 100);
     border-radius: 10px;
     padding: 5px;
     font: italic 12pt "Open Sans";
     color: rgb(100, 100, 0);
    </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string>No new notifications</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
